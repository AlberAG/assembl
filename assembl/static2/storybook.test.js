// @flow
/* eslint-disable */
import initStoryshots from '@storybook/addon-storyshots';
/* eslint-enable */

// Allows taking storybook snapshots for UI testing purpose
initStoryshots();