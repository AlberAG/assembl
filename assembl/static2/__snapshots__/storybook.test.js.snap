// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Button101 danger 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "bottom": 0,
      "display": "flex",
      "left": 0,
      "overflow": "auto",
      "position": "fixed",
      "right": 0,
      "top": 0,
    }
  }
>
  <div
    style={
      Object {
        "margin": "auto",
        "maxHeight": "100%",
      }
    }
  >
    <button
      className="integration button btn btn-lg btn-danger"
      disabled={false}
      onClick={[Function]}
      type="button"
    >
      Custom label
    </button>
  </div>
</div>
`;

exports[`Storyshots Button101 default 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "bottom": 0,
      "display": "flex",
      "left": 0,
      "overflow": "auto",
      "position": "fixed",
      "right": 0,
      "top": 0,
    }
  }
>
  <div
    style={
      Object {
        "margin": "auto",
        "maxHeight": "100%",
      }
    }
  >
    <button
      className="integration button btn btn-lg btn-info"
      disabled={false}
      onClick={[Function]}
      type="button"
    >
      Bluenove
    </button>
  </div>
</div>
`;

exports[`Storyshots Button101 disabled 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "bottom": 0,
      "display": "flex",
      "left": 0,
      "overflow": "auto",
      "position": "fixed",
      "right": 0,
      "top": 0,
    }
  }
>
  <div
    style={
      Object {
        "margin": "auto",
        "maxHeight": "100%",
      }
    }
  >
    <button
      className="integration button btn btn-lg btn-info"
      disabled={true}
      onClick={[Function]}
      type="button"
    >
      Bluenove
    </button>
  </div>
</div>
`;

exports[`Storyshots Checkbox101 checked 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "bottom": 0,
      "display": "flex",
      "left": 0,
      "overflow": "auto",
      "position": "fixed",
      "right": 0,
      "top": 0,
    }
  }
>
  <div
    style={
      Object {
        "margin": "auto",
        "maxHeight": "100%",
      }
    }
  >
    <div
      className="integration checkbox"
    >
      <input
        checked={true}
        id="checkbox"
        onChange={[Function]}
        type="checkbox"
      />
      <label
        htmlFor="checkbox"
      >
        Default
      </label>
    </div>
  </div>
</div>
`;

exports[`Storyshots Checkbox101 checked 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "bottom": 0,
      "display": "flex",
      "left": 0,
      "overflow": "auto",
      "position": "fixed",
      "right": 0,
      "top": 0,
    }
  }
>
  <div
    style={
      Object {
        "margin": "auto",
        "maxHeight": "100%",
      }
    }
  >
    <div
      className="integration checkbox"
    >
      <input
        checked={true}
        id="checkbox"
        onChange={[Function]}
        type="checkbox"
      />
      <label
        htmlFor="checkbox"
      >
        Default
      </label>
    </div>
  </div>
</div>
`;

exports[`Storyshots Checkbox101 custom label 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "bottom": 0,
      "display": "flex",
      "left": 0,
      "overflow": "auto",
      "position": "fixed",
      "right": 0,
      "top": 0,
    }
  }
>
  <div
    style={
      Object {
        "margin": "auto",
        "maxHeight": "100%",
      }
    }
  >
    <div
      className="integration checkbox"
    >
      <input
        checked={false}
        id="checkbox"
        onChange={[Function]}
        type="checkbox"
      />
      <label
        htmlFor="checkbox"
      >
        Custom Label
      </label>
    </div>
  </div>
</div>
`;

exports[`Storyshots Checkbox101 default 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "bottom": 0,
      "display": "flex",
      "left": 0,
      "overflow": "auto",
      "position": "fixed",
      "right": 0,
      "top": 0,
    }
  }
>
  <div
    style={
      Object {
        "margin": "auto",
        "maxHeight": "100%",
      }
    }
  >
    <div
      className="integration checkbox"
    >
      <input
        checked={false}
        id="checkbox"
        onChange={[Function]}
        type="checkbox"
      />
      <label
        htmlFor="checkbox"
      >
        Default
      </label>
    </div>
  </div>
</div>
`;
